sdk/keyvault/mgmt/pom.xml

```diff
@@ -1,135 +0,0 @@
-<!--
- Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
why are we removing this ?

Reply:
That file was accidentally added through a merge from master and was breaking the build.

Reply:
@weshaggard suggested we removed it.